plaidml/edsl/tests/edsl_test.cc

```diff
@@ -386,7 +386,7 @@ TEST_F(CppEdsl, DoubleDot) {
   // clang-format off
   // CHECK-LABEL: CppEdsl.DoubleDot
   // CHECK: func @double_dot
-  // CHECK: tensor<10x20xf32>) -> tensor<10x40xf32> {
+  // CHECK: tensor<10x20xf32>) -> tensor<10x40xf32> attributes {entrypoint} {
```

Comment:
Can you explain why we need a new attribute? There should already be an entry member on the `Program` that we can use to identify and lookup the entrypoint.

Reply:
During lowering, I needed to figure out the name of the function to attach the `Device` parameter to.  OTOH, I didn't try too hard to find an alternative mechanism; I think it's better to have this information in the MLIR than in a side channel.